### PR TITLE
Add docker_stable_tag for staticanalysis/bot

### PR DESCRIPTION
**Pull request by La0**
And support this tag in taskcluster hooks (it was already implemented for docker hub as a mandatory parameter)

### CODE REVIEW

File: lib/please_cli/please_cli/deploy.py [hunk 1]
```diff
@@ -472,20 +478,25 @@ def cmd_TASKCLUSTER_HOOK(ctx,
...
-                )
+            versioned_image_tag = '-'.join(reversed(image[11:-7].split('-', 1)))
+            image_tags = [versioned_image_tag]
+            if docker_stable_tag:
+                image_tags.append(docker_stable_tag)
+
+            for image_tag in image_tags:
+                click.echo(' => Uploading docker image `{}:{}` ... '.format(docker_repo, image_tag), nl=False)
+                with click_spinner.spinner():
+                    please_cli.utils.push_docker_image(
```
Comment (marco-c) [inline lib/please_cli/please_cli/deploy.py]: Does this mean having to do two uploads?

Comment (La0) [reply lib/please_cli/please_cli/deploy.py]: No docker is smart, and reuse the same chunks already uploaded

**Comment by La0** [comment]
The push to staging was OKish (one seemingly unrelated failure): https://tools.taskcluster.net/groups/Gz5YjB3kTum-qnr4LyOfLg

**Comment by La0** [comment]
And the tag is present on the repo ! https://hub.docker.com/r/mozillareleng/services/tags

**Pull request merged**